web-dev-server.config.js

```diff
@@ -21,7 +21,8 @@ const hmrI18nPlugin = {
     if (TRANSLATION_FILE_REGEX.test(context.path)) {
       // language=JavaScript
       context.body += `
-        ;;
+        ;
+        ;
```

Comment:
what is that for?

Reply:
I rolled it back. I can explain later what the existing code does.